### DISCUSSION TITLE
diary/2020/04/26/novice-leader-retrospect/

### DISCUSSION THREAD

**Discussion by zzsza** [zzsza/blog-comments#123] [Comments]
#  초보 개발 팀장의 1년 회고 - 좋은 팀장이 되기 위한 노력들

https://zzsza.github.io/diary/2020/04/26/novice-leader-retrospect/

**Comment by zzsza** [comment]
<p>좋은 경험 공유 감사합니다. :)</p>

>_This comment was made by Disqus. 2020-04-26 02:50:55 Young Jae  Kwon_

**Comment by zzsza** [comment]
<p>성윤님 인사이트를 보고 어떻게 이런 생각을 하지? 했는데 아래에 있는 무수한 자료와 책들을 보고 의문이 풀렸습니다 ㅠㅠ 항상 깨달음을 주시네요!  많은 고민을 하시면서 만들어가는 팀이라는게 느껴집니다. 공유 고맙습니다!</p>

>_This comment was made by Disqus. 2020-04-27 10:28:24 June Kim_

**Comment by zzsza** [comment]
<p>영재님 좋은 경험이라 해주셔서 감사합니다!!! 글 작성하길 잘했다는 생각이 드네요 :)</p>

>_This comment was made by Disqus. 2020-04-27 11:47:50 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>하핫.. 항상 좋게 봐주셔서 감사합니다!</p><p>일단 생각하고 실천하며, 사이드로 책이나 자료들을 많이 보고 개선하는 사이클을 즐기고 있어요 :)<br>모든 것은 상황마다 다르니..! 그 상황을 제일 잘 이해하는게 핵심인 것 같더라구요 /ㅁ/<br>읽어주셔서 감사합니다!</p>

>_This comment was made by Disqus. 2020-04-27 11:48:33 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>와 정말 몰입해서 읽었습니다. <br>저 또한 팀장으로 일하고 있는데요<br>부족한 부분이 많다는 것을 느끼네요 ㅜ_ㅜ..<br>중간에 말씀하신 커뮤니티 생긴다면 꼭 참여해보고 싶습니다 하하<br>귀중한 인사이트 공유해주셔서 감사합니다!</p>

>_This comment was made by Disqus. 2020-04-29 11:27:02 BumJune Kim_

**Comment by zzsza** [comment]
<p>커뮤니티 추후에 한번 만들어봐도 좋겠네요!!!</p><p>댓글 감사합니다!!! :)</p>

>_This comment was made by Disqus. 2020-05-09 10:43:22 Seong Yun  Byeon_

**Comment by zzsza** [comment]
<p>여러 개발팀을 이끄는 리더로 있습니다. 글을 읽고 나니 현재 관리자로서 많이 노력을 못했다는 생각이 드네요.<br>데이터야 놀자 2019 에도 참석했는데요. 팀장님의 역할이 크셨네요.<br>많은 도움이 되었습니다. 저도 커뮤니티 생기면 참여해 보고 싶어요.^^</p>

>_This comment was made by Disqus. 2020-07-02 06:47:50 Jeong Woo from korea_

**Comment by zzsza** [comment]
<p>겨우 개발 3년 차에 인력부족으로 어쩔 수 없이 팀장직을 맡았었어요. PoC위주의 프로젝트였기에 부담이 좀 덜하긴 했지만, 혼자서 자기 일만 해오던 저에게 팀원들을 리더해서 원하는 결과물을 얻어내는게 좀 처럼 쉬운게 아니더라구요(혼자하는게 더 빠르겠네.. 라는 생각이 들만큼). 글 읽고 제가 많이 부족했다는 생각이 듭니다..</p>

>_This comment was made by Disqus. 2020-08-23 03:59:05 김다슬_

**Comment by zzsza** [comment]
<p>와 이런 자료 공유 정말 감사합니다^^</p>

>_This comment was made by Disqus. 2021-04-02 02:30:32 jongsang han_

**Comment by zzsza** [comment]
<p>좋은글 감사드립니다</p>

>_This comment was made by Disqus. 2021-05-13 03:06:01 Kadler Kim_

**Comment by zzsza** [comment]
<p>전임 팀장님이 갑자기 퇴사하시는 바람에 개발 3년차 조금 넘어가는 시점에 반 강제로 팀장직을 맡게되버렸네요. 공감되는 내용이 많고 도움이 되는것 같아 좋습니다.</p>

>_This comment was made by Disqus. 2021-05-22 02:45:49 모영권_

**Comment by zzsza** [comment]
<p>좋은 글 감사드립니다.</p>

>_This comment was made by Disqus. 2021-06-29 05:01:03 IanJang_

**Comment by howyoujini** [comment]
내용이 구체적이라서 더 도움이 될 것 같습니다. 아무래도 이 글은 한번 보고 끝날 것 같지 않네요! 자주 찾아와서 읽고 또 읽어보도록 할께요 :) 좋은 글 감사합니다.
